css/style.css

```diff
@@ -118,7 +118,7 @@ html,body{height:100%;}
 	text-shadow: 3px 3px 10px #000;
 }
 
-#contact, #about {
+#contact, #events, #about {
```

Comment:
Push your oduacm.github.io master to a gh-pages branch so I can see if the events looks better like this